docker/cron/crontab

```diff
@@ -0,0 +1,2 @@
+* * * * * sh php bin/console sylius:remove-expired-carts
+* * * * * sh php bin/console sylius:sylius:cancel-unpaid-orders
```

Comment:
```suggestion
0 */6 * * * sh php bin/console sylius:remove-expired-carts
0 */6 * * * sh php bin/console sylius:sylius:cancel-unpaid-orders
```

Probably it doesn't have to be that frequent. The value I propose is copied from the docs: https://docs.sylius.com/en/1.12/cookbook/deployment/platform-sh.html#add-default-sylius-cronjobs

Reply:
I know why I did'nt see that. I focused on this page: https://docs.sylius.com/en/1.12/cookbook/deployment/cron-jobs.html?highlight=cron#how-to-configure-a-cron-job

Reply:
we should also add Symfony messenger for catalog promotions
```suggestion
* * * * * sh php bin/console sylius:remove-expired-carts
* * * * * sh php bin/console sylius:sylius:cancel-unpaid-orders
* * * * * sh php bin/console messenger:consume main
```

Reply:
👍️ and we can add `--time-limit=60` flag to the `messenger:consume` command to have only one consumer running in the background. However, well configured supervisor would be better in this case, but for now, cron job is better than nothing 😛

Reply:
Valid point about the catalog promotions, but yea The messenger:consume will have separated queue worker. That's my next step :)